lib/pages/DesktopFarm.dart

```diff
@@ -230,24 +227,19 @@ class _DesktopFarmState extends State<DesktopFarm> {
             mainAxisAlignment: MainAxisAlignment.spaceBetween,
             children: <Widget>[
               Text(
-                "Total APY",
+                "Current APR",
                 style: txStyle,
               ),
-              Text("12%", style: txStyle)
+              // Obx(() => Text("${farm.strAPR}%", style: txStyle))
```

Comment:
Do not check in commented old code, just remove it if it is not needed

Reply:
I removed it